src/groundlight/client.py

```diff
@@ -95,24 +95,37 @@ def list_detectors(self, page: int = 1, page_size: int = 10) -> PaginatedDetecto
         obj = self.detectors_api.list_detectors(page=page, page_size=page_size)
         return PaginatedDetectorList.parse_obj(obj.to_dict())
 
-    def create_detector(self, name: str, query: str, config_name: str = None) -> Detector:
-        obj = self.detectors_api.create_detector(DetectorCreationInput(name=name, query=query, config_name=config_name))
+    def create_detector(
+        self, name: str, query: str, *, confidence: float = DEFAULT_CONFIDENCE_THRESHOLD, config_name: str = None
```

Comment:
Should this be called `confidence` or `confidence_threshold`?

Reply:
Threshold makes sense to me - it's more clear.